vision/src/autogluon/vision/predictor/predictor.py

```diff
@@ -409,10 +409,9 @@ def fit(self,
         self._train_classes = train_data.classes
         with warnings.catch_warnings(record=True) as w:
             warnings.simplefilter("always")
-            with MXNetErrorCatcher() as err:
-                self._classifier = task.fit(train_data, tuning_data, 1 - holdout_frac, random_state)
-            if err.exc_value is not None:
-                raise RuntimeError(err.exc_value + err.hint)
+            # TODO: MXNetErrorCatcher was removed because it didn't return traceback
+            #  Re-add once it returns full traceback regardless of which exception was caught
+            self._classifier = task.fit(train_data, tuning_data, 1 - holdout_frac, random_state)
```

Comment:
@zhreshold This was done because previously the error would not show stack trace making it extremely hard to debug where errors are coming from. If you can add that functionality to the error catcher, feel free to add it back in, but we need stack traces to be visible.